Add an advanced page for the Brain Board (#410)
srobo/docs: kit/brain_board/advanced.md

---
layout: page
title: Brain Board - Advanced
---

# Brain Board - Advanced

> When connected to your Brain Board WiFi network, it is possible to access the brain board via a terminal interface. The Brain Board is running Student Robotics OS, a purpose-built Linux distribution that does not have all of the conveniences of a usual Linux system.

Reviewer: ```suggestion
When connected to your Brain Board WiFi network, it is possible to access the Brain Board via a terminal interface. The Brain Board is running Student Robotics OS, a purpose-built Linux distribution that does not have all of the conveniences of a usual Linux system.
```

Reviewer: ```suggestion
When connected to your [Brain Board WiFi](/docs/kit/wifi), it is possible to access the brain board via a terminal interface. The Brain Board is running Student Robotics OS, a purpose-built Linux distribution that does not have all of the conveniences of a usual Linux system.
```

<div class="warning">

> The following information is about advanced features for which we only provide limited support. Proceed at your own risk.

Reviewer: Doe this make the warning more active and less passive?
```suggestion
Proceed at your own risk. We only provide limited support for these advanced features.
```
